openreview/conference/matching.py

```diff
@@ -238,86 +323,51 @@ def _build_config_invitation(self, scores_specification):
                         'default': scores_specification
                     },
                     'aggregate_score_invitation': {
-                        'value': self.conference.get_invitation_id('Reviewing/Aggregate_Score'),
+                        'value': self._get_edge_invitation_id('Reviewing/Aggregate_Score'),
                         'required': True,
                         'description': 'Invitation to store aggregated scores',
                         'order': 9
                     },
                     'conflicts_invitation': {
-                        'value': self.conference.get_invitation_id('Reviewing/Conflict'),
+                        'value': self._get_edge_invitation_id('Reviewing/Conflict'),
                         'required': True,
                         'description': 'Invitation to store aggregated scores',
                         'order': 10
                     },
                     'custom_load_invitation': {
-                        'value': self.conference.get_invitation_id('Reviewing/Custom_Load'),
+                        'value': self._get_edge_invitation_id('Reviewing/Custom_Load'),
                         'required': True,
                         'description': 'Invitation to store aggregated scores',
                         'order': 11
                     },
                     'assignment_invitation': {
-                        'value': self.conference.get_invitation_id('Reviewing/Paper_Assignment'),
+                        'value': self._get_edge_invitation_id('Reviewing/Paper_Assignment'),
                         'required': True,
                         'description': 'Invitation to store paper user assignments',
                         'order': 12
                     },
                     'config_invitation': {
-                        'value': self.conference.get_invitation_id('Reviewing/Assignment_Configuration')
+                        'value': self._get_edge_invitation_id('Reviewing/Assignment_Configuration')
                     },
                     'status': {
                         'default': 'Initialized',
-                        'value-dropdown': ['Initialized', 'Running', 'Error', 'No Solution', 'Complete', 'Deployed']
+                        'value-dropdown': [
+                            'Initialized',
+                            'Running',
+                            'Error',
+                            'No Solution',
+                            'Complete',
+                            'Deployed'
+                        ]
                     }
                 }
             })
         self.client.post_invitation(config_inv)
 
-    def _jaccard_similarity(self, list1, list2):
```

Comment:
why do you move these functions outside of the class?

Reply:
Just trying to follow pylint's suggestions. It's because these functions don't use `self` for anything